R/prism_to_gdrDF.R

```diff
@@ -135,6 +135,8 @@ convert_LEVEL6_prism_to_gDR_input <- function(prism_data_path,
                      by = "column_name")
 
   full_data$value <- pmin(readout_min, 2 ^ full_data$value)
+  full_data <- full_data[!(is.na(name) | is.na(value) | name == "DMSO")]
```

Comment:
Is it not a little magic number?
Please, add comment for future us about `name == "DMSO"`.

Reply:
Removed "DMSO" chunk